src/components/Reports/ReportsFilters/ReportsFilters.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const ReportsFilters = ({ domains, domainSelected, pages, changeDomain }) => {
+  return (
+    <header className="report-filters">
+      <div className="report-wrapper">
+        <h3>
+          <FormattedMessage id="reports_filters.title" />
+        </h3>
+        <p>
+          <FormattedMessage id="reports_filters.description" />
+        </p>
+        <form action="#" className="form-filters">
+          <fieldset className="filter">
+            <label htmlFor="domain">
+              <FormattedMessage id="reports_filters.domain" />
+            </label>
+            <span className="dropdown-arrow" />
+            <select
+              onChange={changeDomain}
```

Comment:
What do you think about extracting the id here? in that way, outside components do not need to know about elements, events, etc, for example:

```suggestion
              onChange={(event) => changeDomain(parseInt(event.target.value)}
```

Reply:
fixed